README.md

```diff
@@ -391,10 +391,15 @@ For example, our `engine` node would send:
 
 ```java
 homie/super-car/engine/$name → "Car engine"
-homie/super-car/engine/$type → "V8"
+homie/super-car/engine/$type → "engine"
 homie/super-car/engine/$properties → "speed,direction,temperature"
 ```
 
+`$type` can be any string. The following predefined node types allow controllers to identify and present a node in a more distinct way:
```

Comment:
I'd suggest to add a level 5 headline. Also please break down lines to one sentence per line.

Reply:
I would also change from "any string" to a [topic id](https://github.com/homieiot/convention#topic-ids), since it makes more sense (after all, a $type is an ID)